lib/core/src/receive_swap.rs

```diff
@@ -369,6 +381,52 @@ impl ReceiveSwapHandler {
         }
     }
 
+    async fn claim_confirmed_lockups(&self, height: u32) -> Result<()> {
+        let receive_swaps: Vec<ReceiveSwap> = self
+            .persister
+            .list_ongoing_receive_swaps(Some(true))?
+            .into_iter()
+            .filter(|s| s.lockup_tx_id.is_some() && s.claim_tx_id.is_none())
+            .collect();
+        info!(
+            "Rescanning {} Receive Swap(s) lockup txs at height {}",
+            receive_swaps.len(),
+            height
+        );
+        for swap in receive_swaps {
+            if let Err(e) = self.claim_confirmed_lockup(&swap).await {
```

Comment:
Do we want to skip these with claim_tx_id set here? Otherwise we will probably worst case log here an error.